src/stories/components/svg/twitter.tsx

```diff
@@ -12,7 +12,7 @@ const Twitter = ({
 }: Props) => {
   return (
     <svg width={width} height={height} viewBox={`0 0 ${width} ${height}`} fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is there a reason for change the viewBox?

Reply:
I thought you were fixing this in your commit weren't you? It's not part of the changes I'm doing, If I make that change we will have a conflict when merging.

Reply:
ok, i will fix my pr, to let without  viewBox